sdk/js/src/types.ts

```diff
@@ -93,6 +93,16 @@ export interface DevCycleOptions {
      * Disables logging of custom events generated by calling .track() method to DevCycle.
      */
     disableCustomEventLogging?: boolean
+
+    /**
+     * Controls the maximum size the event queue can grow to until a flush is forced. Defaults to `100`.
+     */
+    flushEventQueueSize?: number
+
+    /**
+     * Controls the maximum size the event queue can grow to until events are dropped. Defaults to `2000`.
```

Comment:
in the file, it defaults to 1000, should change to 1000
```suggestion
     * Controls the maximum size the event queue can grow to until events are dropped. Defaults to `1000`.
```